book/http_cache.rst

```diff
@@ -163,6 +163,23 @@ kernel::
 The caching kernel will immediately act as a reverse proxy - caching responses
 from your application and returning them to the client.
 
+..caution::
+
+    By default, the caching kernel ignores the ``framework.http_method_override``
```

Comment:
What about: "By default, a kernel based the cache ignores the ``framework.http_method_override``" ?